File: example/javalib/springboot/1-hello-spring-boot/src/com/example/HelloSpringBoot.java
package com.example;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RestController;

@SpringBootApplication
public class HelloSpringBoot {

  public static void main(String[] args) {
    SpringApplication.run(HelloSpringBoot.class, args);
  }

  @RestController
  class HelloController {
    @GetMapping("/")
    public String hello() {
      return "<h1>Hello, World!</h1>;
    }
  }
}

File: example/kotlinlib/web/6-hello-spring-boot/src/com/example/HelloSpringBoot.kt
package com.example

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RestController

@SpringBootApplication
open class HelloSpringBoot

fun main(args: Array<String>) {
    runApplication<HelloSpringBoot>(*args)
}

@RestController
class HelloController {
    @GetMapping("/")
    fun hello(): String = "<h1>Hello, World!</h1>"
}
